setup.cfg

```diff
@@ -31,15 +31,29 @@ per-file-ignores =
     tests/*: B011
 
 [mypy]
+follow_imports = silent
```

Comment:
Looks like this line is unnecessary? The default behavior would work well.

Reply:
Yup, seems to be the case. Removed.